Makefile

```diff
@@ -9,9 +9,9 @@ define fix_import
 	if [[ ${1} =~ "openapi" ]] ; then \
 		rp=".." ; \
 	fi && \
-	sed -i '' "s/from google.api/from $${rp}google.api/g" ${1} && \
-	sed -i '' "s/from openapiv3/from $${rp}openapiv3/g" ${1} && \
-	sed -i '' "s/from server.v1/from $${rp}server.v1/g" ${1}
+	sed -i "s/from google.api/from $${rp}google.api/g" ${1} && \
```

Comment:
this may not work on mac

Reply:
The previous line was failing for me on mac

```
sed: can't read s/from google.api/from ..google.api/g: No such file or directory
make: *** [fix_imports] Error 2
```

Reply:
hmmm... this fails for me... let me dive deeper

```
sed: 1: "/Users/adil/devbox/goco ...": command a expects \ followed by text
make: *** [fix_imports] Error 1
```